fera_full.py

```
@@ -1,3 +1,189 @@
+                if(global_settings.pathdb==None):
+                    return
+                global_settings.initiate_processes()
+                fera.start_fera_app()  
+        else:
+            print(1)
+    except Exception as ex:
+        utilities_general.printlogexception(ex=ex)
+    finally:
+        try:
+            sqliteconn.close()
+        except:
```

Ruff

Do not use bare `except`

(E722)